src/components/Trend/__test__/Trend.test.js

```diff
@@ -79,4 +79,25 @@ describe('Trend', () => {
     expect(lastStopProps.offset).toEqual(1);
     expect(lastStopProps.stopColor).toEqual('red');
   });
+
+  it('handles an array of value objects', () => {
+    // I don't really know the best way to test this, beyond checking that
+    // it doesn't blow up.
```

Comment:
Also that comment was left over from a copy-paster earlier - I made quite a few mistakes in this small PR, lol.